### PR TITLE
feat: add background-color transitions to table color changes

### PR DESCRIPTION
**Pull request by JFrankfurt**
https://user-images.githubusercontent.com/5773490/186736049-98122ea3-15b3-457e-aa6e-92b6828a359f.mov

### CODE REVIEW

File: src/components/Tokens/TokenTable/TokenRow.tsx [hunk 1]
```diff
@@ -41,24 +41,33 @@ const Cell = styled.div`
...
+    ${({ loading, theme }) =>
+      !loading &&
+      css`
+        background-color: ${theme.accentActionSoft};
+      `}
+    ${({ last }) =>
+      last &&
+      css`
+        border-radius: 0px 0px 8px 8px;
+      `}
```
Comment (JFrankfurt) [inline src/components/Tokens/TokenTable/TokenRow.tsx:70]: bigger, but gives us syntax highlighting!

File: src/components/Tokens/TokenTable/TokenRow.tsx [hunk 2]
```diff
@@ -41,24 +41,33 @@ const Cell = styled.div`
...
+  height: 60px;
   line-height: 24px;
-
   max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
   min-width: 390px;
   padding-top: ${({ first }) => (first ? '4px' : '0px')};
   padding-bottom: ${({ last }) => (last ? '4px' : '0px')};
   padding-left: 12px;
   padding-right: 12px;
+  transition: background-color 250ms ease;
```
Comment (vm) [inline src/components/Tokens/TokenTable/TokenRow.tsx:57]: there were some transition variables added to theme so please use those

**Review by vm — APPROVED** [review]
good after the variables change

File: src/components/Tokens/TokenTable/TokenRow.tsx (hunk 2 above)
Comment (JFrankfurt) [reply src/components/Tokens/TokenTable/TokenRow.tsx:57]: ah cool didn't think those'd be in so soon

**Pull request merged**